.circleci/config.yml

```diff
@@ -260,6 +260,34 @@ jobs:
           service: << parameters.service >>
           environment_key: staging
 
+  renew_search_references:
+    parameters:
+      environment_key:
+        type: string
+    docker:
+      - image: cimg/ruby:3.1.2
+    steps:
+      - checkout
+      - tariff/cf-install:
+          space: << parameters.environment_key >>
+      - run:
+          name: "Install necessary dependencies"
+          command: |
+            cf install-plugin conduit -r CF-Community -f
+            sudo apt-get update -qq
+            sudo apt-get install postgresql-client
+      - run:
+          name: "Dump the uk search_references table"
+          command: |
+            cf conduit "tariff-uk-<< parameters.environment_key >>-postgres" -- pg_dump --file tariff-uk-<< parameters.environment_key >>-postgres-search_references.psql --no-acl --no-owner --clean --verbose --table search_references
+      - run:
+          name: "Restore the uk search references into xi"
+          command: |
+            cf conduit tariff-xi-<< parameters.environment_key >>-postgres -- psql $PGDATABASE < tariff-uk-<< parameters.environment_key >>-postgres-search_references.psql
```

Comment:
do you need `--verbose` here? I saw you used it in the previous run

Reply:
It would just create stdout mess. My validations will include verifying that the regression suite passes and that the search references between the two services match on Monday. I'll also check the number of active connections